test/integration/short/metadata-tests.js

```diff
@@ -147,32 +178,39 @@ describe('Metadata', function () {
     });
     vit('2.1', 'should retrieve the updated metadata after a schema change', function (done) {
       var client = newInstance({ refreshSchemaDelay: 50 });
+      var nonSyncClient = newInstance({ isMetadataSyncEnabled: false });
+      var clients = [client, nonSyncClient];
       utils.series([
         client.connect.bind(client),
-        helper.toTask(client.execute, client, helper.createKeyspaceCql('ks_udt_meta', 3)),
+        nonSyncClient.connect.bind(nonSyncClient),
+        helper.toTask(client.execute, client, helper.createKeyspaceCql('ks_udt_meta', 2)),
         helper.toTask(client.execute, client, 'USE ks_udt_meta'),
         helper.toTask(client.execute, client, 'CREATE TYPE type_changing (id uuid, name ascii)'),
-        helper.delay(1000),
```

Comment:
A nice side effect of the change to not complete the request until the schema is updated is we shouldn't have to have these delays anymore, but will see if that is truly the case via CI.